compiler-rt/src/alu/icmp_sge.cairo

```diff
@@ -0,0 +1,35 @@
+pub mod icmp_sge_i8;
+
+use crate::alu::scmp::scmp;
+use core::num::traits::{BitSize, Bounded, OverflowingSub};
+
+// Perform the `icmp_sge` operation.
+//
+// This is a generic implementation for every data type. Its specialized versions
+// are defined and tested in the icmp_sge/icmp_sge_<type>.cairo files.
+fn icmp_sge<
+    T,
+    // The trait bounds are chosen so that:
+    //
+    // - BitSize<T>: we can determine the length of the data type in bits,
+    // - Bounded<T>: we can determine min and max value of the type,
+    // - TryInto<u128, T>, Into<T, u128> - we can convert the type from/to u128,
+    // - Destruct<T>: the type can be dropped as the result of the downcasting check.
+    // - OverflowingSub<T>: so we can use T.overflowing_sub().
+    //
+    // Overall these trait bounds allow any unsigned integer to be used as the concrete type.
+    impl TBitSize: BitSize<T>,
+    impl TBounded: Bounded<T>,
+    impl TTryInto: TryInto<u128, T>,
+    impl TInto: Into<T, u128>,
+    impl TDestruct: Destruct<T>,
+    impl TOverflowingSub: OverflowingSub<T>,
+>(
+    lhs: u128, rhs: u128,
+) -> u128 {
+    match scmp::<T>(lhs, rhs) {
```

Comment:
`assert_fits_in_type` is called in `scmp`. Should there be a comment mentioning that `assert_fits_in_type` is done elsewhere, in case of future changes to the body of the functions?

Reply:
Yeah this should definitely be clear about the behaviour it is relying on from `scmp` here.